xtask/src/utils.rs

```diff
@@ -0,0 +1,99 @@
+use serde_json::Value;
+use std::{process::Command, time::Duration};
+
+pub(crate) enum WorkspaceMemberType {
+    Crate,
+    Example,
+}
+
+#[derive(Debug)]
+pub(crate) struct WorkspaceMember {
+    pub name: String,
+    pub path: String,
```

Comment:
We can use `pub(crate)` or remove the visibility attribute if they are used in the same file